test/desktop_sikuli/tests/test_recover_access.py

```diff
@@ -16,6 +21,7 @@ def test_recover_access(self):
         profile.share_my_code_button.click()
         profile.find_text(base_user['public_key'])
 
+    @pytest.mark.testrail_id(5571)
```

Comment:
This TC includes navigation using "Tab" key, but it is not implemented yet on desktop side.
Should I move it to separate TC?

Reply:
I'd leave it as it is, the test will fail